src/xrAICore/Components/script_world_property_script.cpp

```diff
@@ -20,4 +20,4 @@ SCRIPT_EXPORT(CScriptWorldProperty, (), {
                          .def("value", &CScriptWorldProperty::value)
                          .def(const_self < other<CScriptWorldProperty>())
                          .def(const_self == other<CScriptWorldProperty>())];
-});
+})
```

Comment:
```suggestion
});
```

Reply:
Extra ';` - why?

Reply:
This PR is about bugfixes, not cosmetic changes.

> Котлеты отдельно, мухи отдельно. ©️ Путин